src/marks/tip.js

```diff
@@ -188,8 +188,8 @@ export class Tip extends Mark {
       if (j >= 0) {
         // name is truncated
         name = name.slice(0, j).trimEnd() + ellipsis;
-        value = "";
         title = value.trim();
+        value = "";
```

Comment:
this change is for an unrelated bug, where trimming the label would not result in the title being empty instead of showing the value